lib/workers/repository/update/pr/index.ts

```diff
@@ -57,6 +64,37 @@ export type ResultWithoutPr = {
 
 export type EnsurePrResult = ResultWithPr | ResultWithoutPr;
 
+export function updatePrRenovateVerData(
+  existingPr: Pr | undefined,
+  prBody: string
+): string {
+  let res = prBody;
+  let oldData;
+  let prDataNew: RenovatePrData;
+  if (existingPr) {
+    oldData = existingPr.bodyStruct?.renovatePrVerData;
+    if (oldData) {
+      prDataNew = JSON.parse(oldData);
+      prDataNew.prUpdateVer = pkg.version;
+    } else {
+      prDataNew = { prCreationVer: 'unknown', prUpdateVer: pkg.version };
+    }
+  } else {
+    prDataNew = { prCreationVer: pkg.version, prUpdateVer: pkg.version };
+  }
+
+  const prVerNew64 = toBase64(JSON.stringify(prDataNew));
+
+  if (oldData) {
+    res = res.replace(prVerDataRe, `<!--renovate-pr-data:${prVerNew64}-->`);
+  } else {
+    res += `\n<!--renovate-pr-data:${prVerNew64}-->\n`;
+  }
+
+  logger.debug(`PR Renovate Version Data:${JSON.stringify(prDataNew)}`);
```

Comment:
i think we should use logger meta here

Reply:
like this? `logger.debug(prDataNew, `PR Renovate Version Data`);`